runtime/onert/api/include/nnfw.h

```diff
@@ -202,6 +204,28 @@ NNFW_STATUS nnfw_load_model_from_file(nnfw_session *session, const char *package
 NNFW_STATUS nnfw_apply_tensorinfo(nnfw_session *session, uint32_t index,
                                   nnfw_tensorinfo tensor_info);
 
+/**
+ * @brief    Set input model's tensor info for resizing
+ *
+ * This function can be called in any time after calling {@link nnfw_model_load_from_file}. Changing
+ * input tensor's shape will cause shape inference for the model. There are two different types of
+ * shape inference - static and dynamic. Which one to use is depend on the current state of the
+ * session.
+ * When it is called after calling {@link nnfw_model_load_from_file} and before calling {@link
+ * nnfw_prepare}, this info will be used when {@link nnfw_prepare}. And it will perform static shape
+ * inference for all tensors.
+ * When it is called after calling {@link nnfw_prepare} or even after {@link nnfw_run}, this info
```

Comment:
Does ` even after {@link nnfw_run}` means that this call will make an effect on the next run?

Reply:
Yes!